### PR TITLE
[BACKLOG-42706] - Update plugin to support project UI

### PR DESCRIPTION
**Pull request by abryant-hv**

### CODE REVIEW

File: impl/logs/pdi.log
Comment (bconlon-hv) [inline impl/logs/pdi.log:1]: what's this empty file?

Comment (niehusa) [reply impl/logs/pdi.log:1]: ? +1

Comment (abryant-hv) [reply impl/logs/pdi.log:1]: it's not in .gitignore, so it keeps getting added. 

I will fix

**Review by pdesai16 — APPROVED** [review]

File: impl/logs/pdi.log
Comment (abryant-hv) [reply impl/logs/pdi.log:1]: Removed that file. Someone can merge this?

**Pull request merged**